docs/moderator-onboarding-guide.md

```diff
@@ -0,0 +1,49 @@
+# The Official freeCodeCamp Moderator Onboarding Guide
+
+This guide will help new moderators get up-and-running with the processes and procedures followed by experienced moderators on the fCC community forum and other official communities that we foster.
+
+> [!NOTE]
+> If you haven't read [The Moderator Handbook](https://contribute.freecodecamp.org/#/moderator-handbook) yet, you should start there first.
+
+## The Forum
+
+### First Steps
+
+The first thing you may notice after being given moderator status on the forum is that your interface will look somewhat different with new admin tools to explore and access to the Mod-Team subforum as well.
+
+Some of the new tools will appear inside a new menu item that looks like a wrench. Some will appear as new tabs or buttons or even new enabled options within the forum menus.
+
+To get familiar with the new tools and powers you can combine one or more of the following methods during your first week with this elevated role. (The first two of which are the most important.)
+
+### Become Familiar with the Discourse Admin Tools
+
+The freeCodeCamp forum is actually a Discourse forum and follows many of the same guidelines of other forums built on Discourse. To begin to get familiar with Discourse and the moderation role, start by reading Discourse's [new user guide](https://meta.discourse.org/t/discourse-new-user-guide/96331) then read Discourse's [new moderator guide](https://meta.discourse.org/t/discourse-moderation-guide/63116).
+
+### Shadow a Mod
+
+All moderator actions can be followed by reviewing the [action logs](https://forum.freecodecamp.org/admin/logs/staff_action_logs). The actions taken by automated tools like 'akistmet' or 'system' can mostly be ignored until they result in a post that needs to be reviewed. Posts to be reviewed will show up in the [Review](https://forum.freecodecamp.org/review) area of the forum.
+
+For the first week or so you will want to pay attention to what is getting flagged and what is being reviewed and compare that to the actions being taken upon the flagged posts. So you may see the system account flag a post because the user created it too quickly. In many cases, the moderators will unflag the post by clicking on the "Approve Post" button or mark it as "Not Spam" (depending on the flag type).
```

Comment:
```suggestion
For the first week or so you will want to pay attention to what is getting flagged and what is being reviewed, and compare that to the actions being taken upon the flagged posts. You may see the system account flag a post because the user created it too quickly. In many cases, the moderators will unflag the post by clicking on the "Approve Post" button or mark it as "Not Spam" (depending on the flag type).
```